### PR TITLE
Quick rework.

### PR DESCRIPTION
**Pull request by ypid**

### CODE REVIEW

File: tasks/sentinel.yml [hunk 1]
```diff
@@ -97,28 +101,41 @@
...
+    src: 'etc/init.d/service.j2'
+    dest: '/etc/init.d/redis-{{ item.name }}'
+    owner: 'redis'
+    group: 'redis'
+    mode: '0755'
   with_items:
-    - { name: 'sentinel', config: '{{ redis_sentinel_config }}' }
+    - name: 'sentinel'
+      config: '{{ redis_sentinel_config }}'
   notify: ['Restart Redis Sentinel']
```
Comment (drybjed) [inline tasks/sentinel.yml]: For this to work correctly, you need to reload `systemd` daemons. I prefer this to be a task, something that acts on registered variable, say:
```
- name: Reload systemd daemons
  command: systemctl daemon-reload
  when: (ansible_service_mgr == 'systemd' and redis__register_sentinel_init|changed)
```

File: tasks/server.yml [hunk 2]
```diff
@@ -100,27 +104,40 @@
...
+    src: 'etc/init.d/service.j2'
+    dest: '/etc/init.d/redis-{{ item.name }}'
+    owner: 'redis'
+    group: 'redis'
+    mode: '0755'
   with_items:
-    - { name: 'server', config: '{{ redis_config }}' }
+    - name: 'server'
+      config: '{{ redis_config }}'
   notify: ['Restart Redis Server']
```
Comment (drybjed) [inline tasks/server.yml]: This also requires a reload of `systemd` daemons (see above).

Comment (ypid) [inline tasks/server.yml]: Thanks. Fixed.

**Comment by drybjed** [comment]
@ypid Thanks. You think it's ready for 0.1.0 release? Or do you want to add anything else?

**Comment by ypid** [comment]
Hm, I don’t really think it is ready for a release but I am not working on more rework either.

**Comment by drybjed** [comment]
OK. I'm gonna check if role works on clean and dirty hosts, and if it does I'll tag this so that variables could be updated later, perhaps configuration reworked to use `blockinfile` module, etc. At least there will be a stable release.

**Comment by drybjed** [comment]
Interestingly, at least on Debian Jessie, `redis-server` service is run by a provided `systemd` unit instead via the role. I suppose that's OK, since it uses the same configuration file.

**Comment by ypid** [comment]
OK, then the sysv init script is only there for legacy systems :wink: Thanks for the input.

**Pull request merged**